schemas/test/en/test_list_collector_repeating_blocks_section_summary.json

```diff
@@ -0,0 +1,361 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "title": "Test a List Collector with Repeating Blocks and Section Summary Items",
+    "theme": "default",
+    "description": "A questionnaire to test a list collector with repeating blocks and section summary items",
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "answer_codes": [
+        {
+            "answer_id": "any-companies-or-branches-answer",
+            "code": "1"
+        },
+        {
+            "answer_id": "company-or-branch-name",
+            "code": "1a"
+        },
+        {
+            "answer_id": "registration-number",
+            "code": "1b"
+        },
+        {
+            "answer_id": "registration-date",
+            "code": "1c"
+        },
+        {
+            "answer_id": "authorised-trader-uk-radio",
+            "code": "1d"
+        },
+        {
+            "answer_id": "authorised-trader-eu-radio",
+            "code": "1e"
+        },
+        {
+            "answer_id": "any-other-companies-or-branches-answer",
+            "code": "2"
+        }
+    ],
+    "questionnaire_flow": {
```

Comment:
change one of the tests to use `Hub`? probably this one

Reply:
Not sure if there's much value in this, Hub and repeating blocks wouldn't interact?

Reply:
Just noticed this comment - I think it may be worth changing this to a hub in order to be able to functionally test the completeness of the section when adding new items etc?